Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
```

Comment:
### [rubocop](https://rubocop.org/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.26.1 → 1.35.1

[_[change-log](https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop/rubocop/), [gem-diff](https://my.diffend.io/gems/rubocop/1.26.1/1.35.1)_]





<details>
<summary>Commits</summary>

A change of **497** commits. See the full changes on [the compare page](https://github.com/rubocop/rubocop/compare/v1.26.1...v1.35.1).

These are the first 10 commits:
- (b6c9b2a) [Fix Layout/SpaceInsideBlockBraces for blocks with numbered a…](https://github.com/rubocop/rubocop/commit/b6c9b2ac675315973ef301ce569c70e698007d7b)
- (0d09460) [Merge pull request #10736 from gsamokovarov/space-inside-num…](https://github.com/rubocop/rubocop/commit/0d09460464242c949fa25dd7c116a3d4afac70ff)
- (9876f51) [[Fix #10737] Fix crash in `Style/ConditionalAssignment` with…](https://github.com/rubocop/rubocop/commit/9876f5185c17b585eb9da0ea610d2eb3fc251555)
- (980fb94) [Reduce unused `require_relative` for formatters](https://github.com/rubocop/rubocop/commit/980fb945455d36396a047ca249fe25c13c04807c)
- (c993657) [[Docs] Fix Gemspec/RequireMFA example code spacing](https://github.com/rubocop/rubocop/commit/c9936571ec653da8f304573166041ac14791a84b)
- (340fca4) [Merge pull request #10741 from BrianHawley/fix_require_mfa_s…](https://github.com/rubocop/rubocop/commit/340fca4fdc952f227494a0ddbe176a4c20516242)
- (08833f9) [Merge pull request #10738 from dvandersluis/issue/10737](https://github.com/rubocop/rubocop/commit/08833f9170fc5cdba3138b0ee556b3a6a2d21fb5)
- (8eb4f1b) [[Fix #7900] Fix `Style/FormatStringToken` false positive wit…](https://github.com/rubocop/rubocop/commit/8eb4f1b99ac2c4d28a7aa22ad0e953544f082916)
- (b121da6) [Split a custom rake task into `rubocop:autocorrect` and `rub…](https://github.com/rubocop/rubocop/commit/b121da695dc8dffd3837f850eac12f8439ab2300)
- (92385a5) [[Fix #10700] Update `Style/EmptyMethod` to not correct if th…](https://github.com/rubocop/rubocop/commit/92385a57ff6255e8234101898d2e270abbc8c505)


</details>